docs/develop/create-sso-office-add-ins-nodejs.md

```diff
@@ -28,7 +28,7 @@ This article walks you through the process of enabling single sign-on (SSO) in a
 
 * At least a few files and folders stored on OneDrive for Business in your Office 365 subscription.
```

Comment:
We don't require that files are stored on OneDrive for using the Yo Office generated SSO project because we just user identify data (name, email, office, phone number)

Reply:
@TCourtneyOwen this content exists in the [Node.js tutorial that Ricky wrote previously](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/create-sso-office-add-ins-nodejs?branch=kb-sso-quickstart), which does actually require some OneDrive content. (The SSO quick start -- which uses Yo Office -- is a bit further down in this PR and doesn't mention OneDrive as a prerequisite.)